cpp/leaky_frontend_match.cc

```diff
@@ -22,6 +22,13 @@ Tuple2<Id_t, int> OneToken(lex_mode_t lex_mode, Str* line, int start_pos) {
   return Tuple2<Id_t, int>(static_cast<Id_t>(id), end_pos);
 }
 
+SimpleLexer::SimpleLexer(MatchFunc match_func, Str* s)
+    : Obj(Tag::FixedSize, SimpleLexer::field_mask(), sizeof(SimpleLexer)),
```

Comment:
Hm should we use this `::field_mask()` style everywhere?

It's kind of interesting that you can define it inline